samples/drawing/drawing.cpp

```diff
@@ -822,7 +822,7 @@ void MyCanvas::DrawText(wxDC& dc)
     dc.DrawRotatedText( wxT("That is text"), 20, 10, -45 );
 
     // use wxSWISS_FONT and not wxNORMAL_FONT as the latter can't be rotated
-    // under Win9x (it is not TrueType)
+    // under MSW (it is not TrueType)
```

Comment:
I think what @lanurmi is saying is that the entire comment is no longer necessary, and neither is the actual calls to change the font to `wxSWISS_FONT` (and probably the `wxFONTFAMILY_SWISS` call below that). Does the text rotate fine without these calls on XP+?